erpnext/accounts/doctype/bulk_statement_of_accounts_customers/bulk_statement_of_accounts_customers.json

```
@@ -0,0 +1,47 @@
+{
+ "actions": [],
+ "creation": "2020-05-22 16:48:00.419126",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "customer",
+  "billing_email",
+  "primary_email"
+ ],
+ "fields": [
+  {
+   "fieldname": "customer",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Customer",
+   "options": "Customer",
+   "reqd": 1
+  },
+  {
+   "fetch_from": "this.customers",
+   "fieldname": "billing_email",
+   "fieldtype": "Read Only",
+   "in_list_view": 1,
+   "label": "Billing Email"
+  },
+  {
+   "fieldname": "primary_email",
+   "fieldtype": "Read Only",
+   "in_list_view": 1,
+   "label": "Primary Email"
+  }
+ ],
+ "istable": 1,
+ "links": [],
+ "modified": "2020-07-25 13:08:56.220460",
+ "modified_by": "Administrator",
+ "module": "Accounts",
+ "name": "Bulk Statement Of Accounts Customers",
```

We always use the singular form in doctype naming.

```suggestion
 "name": "Bulk Statement Of Accounts Customer",
```